sematic/db/models/run.py

```diff
@@ -59,15 +59,17 @@ class Run(Base, JSONEncodableMixin, HasExternalJobsMixin):
         The run's description. Defaults to the function's docstring.
     source_code: str
         The calculator's source code.
-    exception: Optional[str]
-        The exception from the calculator's execution
     nested_future_id:
         If the run resulted in returning a new future, this contains the id of that
-        future
+        future.
+    exception_metadata: Optional[ExceptionMetadata]
+        The metadata for the exception from the calculator's execution, if any.
+    external_exception_metadata: Optional[ExceptionMetadata]
```

Comment:
The field names don't have `_metadata` in them.

Reply:
The existing parameter documentation described the `@property`, not the `Column`. What should it be?
<img width="483" alt="image" src="https://user-images.githubusercontent.com/1894533/198355652-7a657ceb-30e9-4d52-b8a0-6dd1bf698709.png">